docs/HowTo/Prevent-Slashing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Import or export a slashing protection file
+---
+
+# Slashing Protection
+
+[Teku implements slashing protection] to prevent validators from signing incorrect
+blocks or attestations.
+
+You can import and export the slashing-protection file when migrating validator keys between
+validator clients.
+
+## Importing a slashing-protection file
+
+When importing the slashing-protection file, Teku imports the file to the
+`<data-path>/validators/slashprotection/` directory in the format `<validator-pubkey>.yml`
+(with no 0x prefix).
+
+Teku supports files using the [Minimal] or [Complete] interchange format when importing.
+
+!!! example
+
+    ```bash
+    teku slashing-protection import --data-path=/home/me/me_node --from=/home/slash/validatorNode1.yml
```

Comment:
importing is from the interchange format, which is json and for potentially a large number of validators in the one import file... `--from=/home/slash/slashing-interchange-format.json`

Reply:
Done